client/src/components/advancedSearch/Condition.tsx

```diff
@@ -0,0 +1,271 @@
+import React, { useState, useEffect, FC } from 'react';
+import PropTypes from 'prop-types';
+import {
+  makeStyles,
+  Theme,
+  createStyles,
+  IconButton,
+  TextField,
+  MenuItem,
+} from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+
+interface ConditionProps {
+  others?: object;
+  onDelete: () => void;
+  triggerChange: (data: TriggerChangeData) => void;
+  fields: Field[];
+  id: string;
+  initData: any;
+  className?: string;
+}
+
+interface Field {
+  name: string;
+  type: 'int' | 'float';
+}
+
+interface TriggerChangeData {
+  field: Field;
+  op: string;
+  value: string;
+  isCorrect: boolean;
+  id: string;
+}
+
+// Static logical operators.
+const LogicalOperators = [
```

Comment:
We have a const folder, may be we can move static values to there.

Reply:
Will do after schema pages merged.